packages/atomic/src/components/common/facets/category-facet/child-value-link.tsx

```diff
@@ -0,0 +1,18 @@
+import {FunctionalComponent, h} from '@stencil/core';
+import {
+  CategoryFacetValueLink,
+  CategoryFacetValueLinkProps,
+} from './value-link';
+
+interface CategoryFacetParentValueLinkProps
```

Comment:
I believe this interface should be renamed to `CategoryFacetChildValueLinkProps`